app/assets/sass/modules/_web-payments.scss

```diff
@@ -1,20 +1,35 @@
-#apple-pay-container,
-#google-pay-container {
+.apple-pay-container{
   display: none;
-}
 
-// Toggle Apple Pay
-.apple-pay-available {
-  #apple-pay-container {
+  .apple-pay-available & {
     display: block;
   }
 }
 
-// Toggle Google Pay
-.google-pay-available {
-  #google-pay-container {
+.google-pay-container {
+  display: none;
+
+  .google-pay-available & {
     display: block;
-  }
+  } 
+}
+
+.web-payment-button-heading{
+  display: none;
+  
+  .apple-pay-available &,
+  .google-pay-available & {
+    display: block; 
+  } 
+}
+
+.non-web-payment-button-heading{
+  display: block;
+
```

Comment:
Opposite from above.  This class is added to Headings that need to be hidden when either Apple or Google Pay button is available.
Shown by default.